ModelBundle/EventListener/ContentTypeOrderFieldListener.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
@alavieille : to avoid the use of a doctrine listener, what do you think of a bundle like this one ? https://github.com/sergeyfedotov/symfony-sortable-collection-type-bundle